src/os/bluestore/BitAllocator.h

```diff
@@ -493,6 +495,7 @@ class BitAllocator:public BitMapAreaIN{
   int64_t alloc_blocks_dis(int64_t num_blocks, int64_t *block_list);
   void free_blocks_dis(int64_t num_blocks, int64_t *block_list);
 
+  int64_t get_truncated_blocks() { return truncated_blocks; }
```

Comment:
This is not required. I have fix for this problem in my next pull request. So no need to keep track of truncated blocks.